chart/templates/core-dns-custom.yaml

```diff
@@ -5,6 +5,5 @@ metadata:
   namespace: kube-system
 data:
   uds.override: |
-    rewrite stop {
-      name regex (.*\.uds\.dev) host.k3d.internal answer auto
-    }
+{{ .Values.coreDnsDefaults | nindent 4 }}
+{{ .Values.coreDnsOverrides | nindent 4 }}
```

Comment:
```suggestion
{{ .Values.coreDnsDefaults | indent 4 }}
{{ .Values.coreDnsOverrides | indent 4 }}
```

Reply:
See https://github.com/defenseunicorns/uds-k3d/pull/112#discussion_r1778889070

Reply:
I decided to remove the Zarf variables altogether for now

Reply:
Working now with this fix!